index.html

```diff
@@ -14,6 +14,9 @@
     />
   </head>
   <body>
+    <div class="popup">
```

Comment:
- [ ] Please The content in your popup should match the content in the card the user select
![image](https://user-images.githubusercontent.com/74744787/226134190-f07451b7-d35f-4970-93f0-246bc1bf4c7b.png)